src/containers/registrationform.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import './style.css';
+import jquery  from "jquery";
+
+
+class RegisterForm extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            fields: {},
+            errors: {}
+        }
+
+        this.handleChange = this.handleChange.bind(this);
+        this.submituserRegistrationForm = this.submituserRegistrationForm.bind(this);
+
+    };
+
+    handleChange(e) {
+        let fields = this.state.fields;
+        fields[e.target.name] = e.target.value;
+        this.setState({
+            fields
+        });
+
+    }
+
+    submituserRegistrationForm(e) {
+        e.preventDefault();
+    
+        if (this.validateForm()) {
+            let field = {};
+            console.log(field);
+            field["username"] = "";
+            field["emailid"] = "";
+            field["mobileno"] = "";
+            field["password"] = "";
+            //this.setState({ fields: field });
+            console.log(JSON.stringify(field["username"]));
+
+            alert("Form submitted");
+        }
+
+       // alert(this.fields["username"]);
+
+        var data = {"name": "Vaarsha", "email":"Varsha@gmail.com", "mobile": "123456", "password": "1234"};
+
+        jquery.ajax({
```

Comment:
Never ever do this.

You have used a heavy package only for AJAX call!! Use `axios` itself.

Reply:
When you use axios, you don't have to worry about any status code.